src/motion/functionality/position.ts

```diff
@@ -103,6 +104,7 @@ export const Position: FunctionalComponentDefinition = {
             values,
             positionTransition,
         }: FunctionalProps) => {
+            useContext(SharedLayoutContext)
```

Comment:
Why is the result from this context not used?

Reply:
When you use a context you can also be saying "update me when this context updates" - the SharedLayoutContext is just updating so this renders as a result